app/src/main/java/com/android/sample/model/recipe/Recipe.kt

```diff
@@ -0,0 +1,12 @@
+package com.android.sample.model.recipe
+
+data class Recipe(
+    val idMeal: Long,
+    val strMeal: String,
+    val strCategory: String,
+    val strArea: String,
```

Comment:
For line 6-7, we don't need to require the category or area of the recipe. It would be better to declare them as String? so that it stays optional.